sorc/orog_mask_tools.fd/inland.fd/inland.F90

```diff
@@ -83,6 +85,8 @@ SUBROUTINE mark_global_inland(cs_res)
 
 END SUBROUTINE mark_global_inland
 
+!> Create inland mask for regional grid.
+!! @param[in] cubed sphere resolution (48, 96...)
```

Comment:
Author tag for every subroutine/function.

Blank doc line between text and first param, like this:

```
!> Some brief description.
!!
!! @param[in] a description of a
!! @author Freddy Fortran
```